lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -443,15 +443,21 @@ balanceTransaction
             _someOtherError ->
                 False
 
--- | Increases the ada value of any 0-ada outputs in the transaction to the
--- minimum according to 'computeMinimumCoinForTxOut'.
-increaseZeroAdaOutputs
+-- | Assigns minimal ada quantities to outputs without ada.
+--
+-- This function only modifies outputs with ada quantities of zero. Outputs
+-- that have non-zero ada quantities will not be modified.
+--
+-- Minimal ada quantities are computed with the 'computeMinimumCoinForTxOut'
+-- function.
+--
+assignMinimalAdaQuantitiesToOutputsWithoutAda
```

Comment:
I was thinking some more about this name, and then realised we _could_ be influenced by the `cardano-ledger-api`'s names in the future: https://github.com/input-output-hk/cardano-ledger/blob/bf2dc0fdde313723b4f7a35362839920670e67ea/libs/cardano-ledger-api/src/Cardano/Ledger/Api/Tx/Out.hs#L119-L120 (seems to have different semantics though)